crates/bevy_gltf/src/loader.rs

```diff
@@ -365,6 +368,7 @@ fn load_node(
                     projection_matrix: perspective_projection.get_projection_matrix(),
                     ..Default::default()
                 });
+                // FIXME how to differentiate between CAMERA_XR and CAMERA_3D?
```

Comment:
I'm not entirely sure what is best here, but I believe that the UI camera differentiation just operates off of names right now.